### PR TITLE
[TWINFACES-58] Edit twinClass link

### PR DESCRIPTION
**Pull request by kerim-alcosi**

### CODE REVIEW

**Review by pihaizer — CHANGES_REQUESTED** [review]
Please use function instead of const where possible. Consider moving some refactoring (like experimental crud data table) to a separate branch,

File: src/entities/twinClassLink/components/resource-link/resource-link.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { TwinClassContext } from "@/app/twinclass/[twinClassId]/twin-class-context";
+import { TwinClass } from "@/lib/api/api-types";
+import { ResourceLink } from "@/shared/ui";
+import { Link2 } from "lucide-react";
+import { useContext } from "react";
+import { TwinClassLinkResourceTooltip } from "./tooltip";
+
+type Props = {
+  data: TwinClass;
```
Comment (kerim-alcosi) [inline src/entities/twinClassLink/components/resource-link/resource-link.tsx:9]: check if `data: TwinClass` is valid

File: src/components/base/data-table/data-table.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ export interface DataTableProps<TData, TValue> {
   disablePagination?: boolean;
   pageSizes?: number[];
   onFetchError?: (e: Error) => any;
-  onRowClick?: (row: TData) => any;
+  onRowClick?: (row: TData) => void;
```
[the remark this answers is not in the document]
Comment (kerim-alcosi) [reply src/components/base/data-table/data-table.tsx]: returned `any`

**Comment by kerim-alcosi** [comment]
> Please use function instead of const where possible.

✅ Done 

> Consider moving some refactoring (like experimental crud data table) to a separate branch,

✅ Excluded refactoring from this PR and moved to draft [PR](https://github.com/alcosi/twinfaces/pull/32) which can be used later in future

**Comment by kerim-alcosi** [comment]
> Please use function instead of const where possible. Consider moving some refactoring (like experimental crud data table) to a separate branch,

https://github.com/alcosi/twinfaces/pull/26#issuecomment-2434793532

**Review by pihaizer — APPROVED** [review]

**Pull request merged**